core/environment/transition_configure.go

```diff
@@ -183,9 +183,11 @@ func (t ConfigureTransition) do(env *Environment) (err error) {
 		// err = t.taskman.ConfigureTasks(env.Id().Array(), tasks)
 		taskmanMessage := task.NewEnvironmentMessage(taskop.ConfigureTasks, env.Id(), tasks, nil)
 		t.taskman.MessageChannel <- taskmanMessage
-		// if err != nil {
-		// 	return
-		// }
+	}
+	incomingEv := <-env.stateChangedCh
```

Comment:
This is a block point... If I understand correctly we're waiting for the overall update of all task state changes relative to this environment, that follows the `ConfigureTasks` operation.

What if an update never comes?